csa/connset.py

```diff
@@ -918,8 +916,7 @@ def iterator (self, low0, high0, low1, high1, state):
         for c in self.subMask.iterator (low1, high1, low0, high0, \
                                         self.transposedState):
             ls.append ((c[1], c[0]))
-        ls.sort (cmpPostOrder)
-        return iter (ls)
+        return iter (sorted(ls))
```

Comment:
I have two problems with this: 1. It doesn't give us post order. 2. It unnecessarily allocates a new list.

Reply:
fixed